zip-0321.rst

```diff
@@ -91,12 +91,14 @@ The following syntax specification uses ABNF [#RFC5234]_.
 
   zcashurn        = "zcash:" ( zcashaddress [ "?" zcashparams ] / "?" zcashparams )
   zcashaddress    = 1*( ALPHA / DIGIT )
+  zcashaddrbundle = zcashaddress [ ";" zcashaddrbundle ]
```

Comment:
```suggestion
  zcashaddrbundle = zcashaddress [ "+" zcashaddrbundle ]
```
I suggest making this change so that a bundle, converted to uppercase, falls within the [Alphanumeric subset](https://en.wikipedia.org/wiki/QR_code#Storage) of a QR code. The intersection of characters in that subset and in `allowed-delims` is `"$" / "*" / "+"`, of which `+` seems the most semantically appropriate. (`:` is also a possibility, but that's in `qchar` rather than `allowed-delims`.)